CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+## Welcome!
+
+We're so glad you're thinking about contributing to an 18F open source project!
+If you're unsure or afraid of anything, just ask or submit the issue or pull
+request anyways. The worst that can happen is that you'll be politely asked to
+change something. We appreciate any sort of contribution, and don't want a
+wall of rules to get in the way of that.
+
+Before contributing, we encourage you to read our CONTRIBUTING policy
+(you are here), our LICENSE, and our README, all of which should be in this
+repository. If you have any questions, or want to read more about our
+underlying policies, you can consult the 18F Open Source Policy GitHub
+repository at https://github.com/18f/open-source-policy, or just shoot us an
+email/official government letterhead note to [18f@gsa.gov](mailto:18f@gsa.gov).
+
+## Contributing to the 18F projects dashboard
+
+If you're reporting an issue, or submitting a fix, please make sure that the
```

Comment:
is this meant to indicate that people should not fix code that was generated by Dashing? Seems like that would be OK